presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeWindowQueries.java

```diff
@@ -115,4 +115,11 @@ public void testRowNumberWithFilter()
         assertQuery("SELECT sum(rn) FROM (SELECT row_number() over() rn, * from orders) WHERE rn = 10");
         assertQuery("SELECT * FROM (SELECT row_number() over(partition by orderstatus order by orderkey) rn, * from orders) WHERE rn = 1");
     }
+
+    @Test
+    public void testWindowFrameBound() {
+        for (int i = 0; i < 10; ++i) {
```

Comment:
@aditi-pandit I find that it requires a few run to make it fail. Any smaller tests you have in mind that we can add as e2e and also reproduce always instead of the loop will be great :)

Reply:
@amitkdutta : https://github.com/prestodb/presto/pull/19847 also hit the issue. 

https://github.com/facebookincubator/velox/pull/5570 should land in a few hours. Lets go with 19847 since that covers many other cases also.

Reply:
Thanks @aditi-pandit. Make sense. Lets go with https://github.com/prestodb/presto/pull/19847/.